_specs/ecip-proto-treasury-system.md

```diff
@@ -0,0 +1,450 @@
+
+Funds will be distributed as follows:
+
+    gitcoinFunds = treasuryBlockReward * gitcoinRewardPerc
+    clientFunds =  (treasuryBlockReward - gitcoinReward) / N 
+
+_Note that if the division by N might has some leftover dust, it will be "collected" by future distributions._
+
+So, for example for the initial proposed state, on which we have three client members, the distribution will be:
+
+- *Gitcoin grant*: 10%
+- *Client Member 1*: 30%
+- *Client Member 2*: 30%
+- *Client Member 3*: 30%
+
+
+Note that as the funds are allocated directly on the treasury contract address, a transaction will be needed to actually distribute ( *distributeFunds()*) them and make them available for withdrawal to each of the members, then each one will need an extra call to do so ( *withdrawFunds()*). 
+As *distributeFunds* considers the state of the contract at that moment, it’s important that any method that alters this state, first executes it.
+
+#### Governance of p-ECTS
+
+##### Proposal and voting mechanics
+
+Governance participation is open to all ETC holders who can vote on any proposed changes to the p-ECTS. 
+
+Votes will be counted as total locked ETC that has participated (voted yes/no) on each proposal.
+
+Anyone can submit a new proposal to update the p-ECTS, but they need to include a minimum amount of funds (*proposalMinDeposit*), those funds will be locked until the proposal ends (not *active*) and will be counted as *yes* votes.
+
+A similar lockdown mechanism applies for the voting, the voter will need to lock their funds for *lockedWaitingTime* until they're elegible and they can cast any vote with them. If the same user wants to register more funds with the same address, all their previous funds will be locked for the new *lockedWaitingTime* period as well.
+
+Participants will be able to unlock these funds and take them back, as long as they are not currently being used as votes on another proposal in progress.
+
+Participants can use the same eligible stake for voting simultaneously on any proposals under consideration.
+
+Participants can change their vote at any time during the proposal period, but all their funds will be taken into account and they will all need to be unlocked. For example, if I vote *yes* with 10ETC, then lock an extra 5ETC, wait the *lockedWaitingTime* and now re-vote as *no*, this will count as a 15ETC *no* vote.
+
+Participants, with the exception of the proposal submitter, can also unregister votes, and the same locking funds restrictions apply. For example, if I vote *yes* with 10ETC, then lock an extra 5ETC, wait the *lockedWaitingTime* and now unregister my votes, naturally, only my 10ETC *yes* vote will be revoked. The funds however, will remain blocked for withdrawl until the proposal finihed.
+
+A proposal lifecycle for updates to p-ECTS consists of the following phases, delimited with certain events:
```

Comment:
nit: the following list does not render as list on github.

Reply:
Yes, sorry about that, should be fine now.